cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -151,6 +151,16 @@ write_files:
         done;
         export HOME="${!HOME_BAK}"
       }
+
+      function measure_startup_time
```

Comment:
the function name should be something like "publish_startup_time" to match what it is doing?